FoodOrderingApp-service/src/main/java/com/upgrad/FoodOrderingApp/service/entity/RestaurantItemEntity.java

```diff
@@ -0,0 +1,55 @@
+package com.upgrad.FoodOrderingApp.service.entity;
+
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "restaurant_item")
+@NamedQueries(
+        {
+
+        }
+)
+public class RestaurantItemEntity implements Serializable {
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @ManyToOne
+    @OnDelete(action = OnDeleteAction.CASCADE)
+    @JoinColumn(name = "item_id")
+    private com.upgrad.FoodOrderingApp.service.entity.ItemEntity item;
```

Comment:
instead of writing this we replace with
private ItemEntity item and same for restaurantentity